src/native/external/zstd-version.txt

```diff
@@ -0,0 +1,11 @@
+Zstandard v1.5.7
+Commit: f8745da6ff1ad1e7bab384bd1f9d742439278e99
+URL: https://github.com/facebook/zstd/releases/tag/v1.5.7
+
+This is the Zstandard v1.5.7 library, released on February 19, 2024. 
+The source code was downloaded from the official GitHub repository and extracted.
+Only the 'lib' and 'build' directories from the original source were included to minimize unnecessary code.
+
+Following patches were applied on top of the 1.5.7 sources:
+- zstd.patch - fixes build issues of CLR on Windows by suppressing enabling of ASM language by CMAKE. zstd contains ASM sources only for Linux
+- zstd-qsort.patch -  based on commit a1a5154b69ddc23cd610e931ee9c6f45884fa9dd to address build issues on Android platform. This patch, together with ZSTD_USE_C90_QSORT define can be removed once we upgrade to 1.5.8.
```

Comment:
nit: we are using this format in other version files:
```suggestion
v1.5.7
https://github.com/facebook/zstd/releases/tag/v1.5.7

Only the 'lib' and 'build' directories from the original source were included to minimize unnecessary code.

Apply https://github.com/facebook/zstd/commit/0d1f8de9addeb2fcb53f1a6a68ef4ac505253c0e
Apply https://github.com/facebook/zstd/commit/ebfa660b8297170fb0742ac8959aa64940292853
Apply src/native/external/zstd.patch # fixes build issues of CLR on Windows by suppressing enabling of ASM language by CMAKE. zstd contains ASM sources only for Linux
```

We should upstream zstd.patch and put a link here.

Reply:
Upstreaming is in progress, will update once merged